Answers.md

```diff
@@ -1 +1,19 @@
 
+#Describe the biggest difference between .forEach & .map.
+So both methods will iterate through each item in an array but the biggest difference is .forEach on its own preforms exactly like a for loop. It Means that an array has the possibility of being mutated. .map on the other hand will return a new array so the original array stays intact.
+
+#What is the difference between a function and a method?
+While all functions are technically methods. A method is a property of an object that has the value of a function.
+
+#What is closure?
+A function is a closure. It is where a local scope is created the the global scope doesn't have access to, but the function has the ability to reach outside of its own scope to grab variables from the global scope. It's quite simple :)
```

Comment:
Almost, but it's about nested functions having access to variables in the outer functions. Otherwise spot on :)